avatars/avatars.js

```diff
@@ -1749,6 +1426,7 @@ class Avatar {
 
       if (this.velocity.length() > maxIdleVelocity) {
         this.lastMoveTime = now;
+        this.move = true;
```

Comment:
If this is a new avatars property, it should be in the constructor, and documented there.